src/tests/qualifications/qualification.test.js

```diff
@@ -0,0 +1,55 @@
+const expect = require('expect')
+const { request } = require('../../utils/test')
+const { QualificationFactory } = require('../../utils/factories/')
+const Qualification = require('../../models/qualifications')
+
+describe('qualifications', () => {
```

Comment:
I'm not so sure what these tests are doing as all of the core mutations have already been tested. These tests are just simply checking if all of the items have been created.

The qualifications tests should check whether a qualification can be added to a user, removed from a user. This should also test retrieving all of the users having a specific qualification (eg how many users have a 'Development' skill).

Reply:
it is signup, adding/removing qualification will go in profile ???

Reply:
Well, you introduced the entity, i assume it's part of it. Doesn't matter if it's in the profile or wherever - a mutation needs to be added to allow this

Reply:
there is a mutation automatically being generated for adding, removing. It can be queried easily from front end.

Reply:
Sorry I must've made you confused, What I mean was we need to check if a certain qualification has been added correctly to a user